arxivdigest/core/config.py

```diff
@@ -35,6 +35,7 @@ def find_config_file(file_locations):
 with open(config_file) as file:
     config = json.load(file)
 
+webaddress_config = config.get('webaddress')
 sql_config = config.get('sql_config')
 email_config = config.get('email_config')
 api_config = config.get('api_config')
```

Comment:
it'd be nicer to call these config_* so that whenever you use these elsewhere in the code, it's more clear immediately that these values come from the config file